### ISSUE TITLE
Start a game release

### ISSUE THREAD

**Issue by andrew-codes** [andrew-codes/playnite-web#573]
Users should be able to start a game release from Playnite Web. This functionality requires a user to be authenticated.

## UX

- [x] Users that are not authenticated do not see a start, stop, or restart buttons in the game details pane.
- [ ] Authenticated users can start a game release from the game details pane
- [x] A game can be started on a specific platform and source combination; e.g. `PlayStation 5` or `PC via Steam`.
- [x] Viewing a game's details pane for a running game, shows the game is running via providing Stop and Restart buttons
    - Note these buttons do not need to do anything for this issue. That will be covered in a separate issue. The purpose is to ensure querying games, or a game can return the runtime state from the graph API.

## Graph API

- [x] Given a game release's ID, platform (name and ID), a game activation (start) request is performed in the form of an internal MQTT message. This internal MQTT message is picked up by the Playnite Web plugin.
    - [x] The release is only processed if the release and platform is found in that Playnite instance.
    - [x] If the release/platform is found and is a computer platform, then the game is started from the Playnite plugin.
- [x] Graph API provides a subscription for the currently playing game's runtime state, including the releases' ID, name, game ID, and process ID (if available).
- [x] Querying for a game or games can include the runtime state of games.

### Extension

- [x] Starting an already installed game will start the game 
- [x] Staring a game that is not installed, will install the game. Allows users to start once the installation is complete.
- [x] Installing a game installs the game
- [x] Uninstall a game uninstalls the game
